components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -312,65 +288,24 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
-                                    <div className="mt-3 flex space-x-1 text-gray-400 dark:text-gray-500">
-                                        <Check className="m-0.5 w-5 h-5 text-orange-500" />
-                                        <div className="flex flex-col">
-                                            <span className="font-bold text-gray-500 dark:text-gray-400">
-                                                Pay-as-you-go after 1,000 credits
-                                            </span>
-                                            <span>
-                                                {currency === "EUR" ? "€" : "$"}0.36 for 10 credits or 1 hour of
-                                                Standard workspace usage, excluding VAT.
-                                            </span>
-                                        </div>
-                                    </div>
-                                </>
-                            ) : (
-                                <>
-                                    <div className="mt-1 text-xl font-semibold flex-grow text-gray-800 dark:text-gray-100">
-                                        Pay-as-you-go
-                                    </div>
-                                    <div className="mt-4 flex space-x-1 text-gray-400 dark:text-gray-500">
-                                        <Check className="m-0.5 w-5 h-5 text-orange-500" />
-                                        <div className="flex flex-col">
-                                            <span>
-                                                {currency === "EUR" ? "€" : "$"}0.36 for 10 credits or 1 hour of
-                                                Standard workspace usage, excluding VAT.{" "}
-                                                <a
-                                                    className="gp-link"
-                                                    href="https://www.gitpod.io/docs/configure/billing/usage-based-billing"
-                                                >
-                                                    Learn more about credits
-                                                </a>
-                                            </span>
-                                        </div>
-                                    </div>
-                                </>
-                            )}
+                            <div className="mt-1 text-xl font-semibold flex-grow text-gray-800 dark:text-gray-100">
+                                Pay-as-you-go
+                            </div>
+                            <div className="mt-4 flex space-x-1 text-gray-400 dark:text-gray-500">
+                                <Check className="m-0.5 w-5 h-5 text-orange-500" />
+                                <div className="flex flex-col">
+                                    <span>
+                                        {priceInformation}{" "}
```

Comment:
**question**: I know this is coming from elsewhere, but could we drop the two decimals (`$9.00`) on the monthly price? Added also some more typo nitpicks if that's possible.

```
// BEFORE
$9.00/month for the first 1000 credits. Afterward pay-as-you-go for $0.036 per credit.

// AFTER
$9 per month for the first 1,000 credits and pay-as-you-go after that for $0.036 per credit.
```

Reply:
please suggest that [here](https://github.com/gitpod-io/ops/issues/7904)

Reply:
Copied the comment in https://github.com/gitpod-io/ops/issues/7904#issuecomment-1398313688. Resolving this. 🏓